From 2ee29d0df3149f05b14b7124e591c32715a6c8d5 Mon Sep 17 00:00:00 2001
From: "Valery V. Vorotyntsev" <valery_vorotyntsev@xyratex.com>
Date: Fri, 20 Jul 2012 12:19:06 +0300
Subject: [PATCH 260/370] confc.h: make subsection headers fit 80 columns

+ There is no such thing as c2_confc_open_async(). Fix invalid identifier
  in the documentation.

RB: r/865/diff/6/?file=24469#file24469line223
---
 conf/confc.h | 10 ++++++----
 1 file changed, 6 insertions(+), 4 deletions(-)

diff --git a/conf/confc.h b/conf/confc.h
index 8479f62..484926d 100644
--- a/conf/confc.h
+++ b/conf/confc.h
@@ -151,11 +151,11 @@ struct c2_mutex;
  * <hr> <!------------------------------------------------------------>
  * @section confc-fspec-recipes Recipes
  *
- * Configuration objects can be opened asynchronously (c2_confc_open_async())
+ * Configuration objects can be opened asynchronously (c2_confc_open())
  * or synchronously (c2_confc_open_sync()). Most of the examples below
  * use synchronous call for the sake of brevity.
  *
- * @subsection confc-fspec-recipe1 Getting data on the filesystem
+ * @subsection confc-fspec-recipe1 Getting configuration data of the filesystem
  *
  * @code
  * #include "conf/confc.h"
@@ -228,7 +228,8 @@ struct c2_mutex;
  * @endcode
  *
  * <!---------------------------------------------------------------->
- * @subsection confc-fspec-recipe2 Getting configuration of a service of specific type
+ * @subsection confc-fspec-recipe2 Service configuration
+ * Getting configuration of a service of specific type.
  *
  * @code
  * #include "conf/confc.h"
@@ -268,7 +269,8 @@ struct c2_mutex;
  * @endcode
  *
  * <!---------------------------------------------------------------->
- * @subsection confc-fspec-recipe3 List devices used by specific service on specific node
+ * @subsection confc-fspec-recipe3 List devices
+ * List devices used by specific service on specific node.
  *
  * @code
  * #include "conf/confc.h"
-- 
1.8.3.2

